flink-table/flink-table-planner/src/test/resources/explain/filesystem/testFileSystemTableSinkWithParallelismInStreamingSql0.out

```diff
@@ -15,9 +15,9 @@ Sink(table=[default_catalog.default_database.test_sink_table], fields=[id, real_
 {
   "nodes" : [ {
     "id" : ,
-    "type" : "Source: TableSourceScan(table=[[default_catalog, default_database, test_source_table]], fields=[id, real_col, double_col, decimal_col])",
+    "type" : "Source: test_source_table[1]",
```

Comment:
replace the node id to avoid the unstable risk ?

Reply:
id removed  when validating in the test